lib/plugins.js

```diff
@@ -131,4 +132,9 @@ Plugins.prototype.teardown = pluginFunFactory('teardown');
  */
 Plugins.prototype.postResults = pluginFunFactory('postResults');
 
+/**
+ * Called after each test block completes.
+ */
+Plugins.prototype.postTest = pluginFunFactory('postTest');
```

Comment:
Thee word "test" seems ambiguous.  Is this after each `it()`, `assert()`, or `describe()`?